src/views/blocks/rsvp/form/form.php

```diff
@@ -8,20 +8,27 @@
  *
  * See more documentation about our Blocks Editor templating system.
  *
- * @link {INSERT_ARTICLE_LINK_HERE}
+ * @link    {INSERT_ARTICLE_LINK_HERE}
  *
- * @since 4.9
- * @since 4.12.0 Add $post_id to filter for hiding opt-outs.
- *
- * @version 4.12.0
+ * @since   4.9
+ * @since   4.12.0 Add $post_id to filter for hiding opt-outs.
+ * @since   TBD Add comments to help IDE auto-completion. Array short syntax.
  *
+ * @version TBD
  */
-$ticket_id   = $this->get( 'ticket_id' );
-$post_id     = $this->get( 'post_id' );
-$going       = $this->get( 'going' );
-$ticket_data = tribe( 'tickets.handler' )->get_object_connections( $ticket_id );
-$event_id    = $ticket_data->event;
-$must_login  = ! is_user_logged_in() && tribe( 'tickets.rsvp' )->login_required();
+
+$ticket_id = $this->get( 'ticket_id' );
+$post_id   = $this->get( 'post_id' );
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $post_id

<sub>hash: 36a7431f3b07c273fbf489e6ad79088e</sub>